packages/blitz/Cargo.toml

```diff
@@ -18,6 +18,7 @@ blitz-renderer-vello = { path = "../blitz-renderer-vello" }
 blitz-html = { path = "../blitz-html" }
 blitz-shell = { path = "../blitz-shell" }
 blitz-net = { path = "../blitz-net", optional = true }
+blitz-traits = { path = "../blitz-traits" }
```

Comment:
@nicoburns not sure this is okay for you. But I am wondering how the net feature cfg could even compile, as it does an import of blitz-traits...

I mean this:

`#[cfg(not(feature = "net"))]
    let net_provider = {
        use blitz_traits::net::DummyNetProvider;
        Arc::new(DummyNetProvider::default())
    };`

Reply:
I think this probably worked because the other `blitz-*` crates depend on `blitz-traits`. But there definitely should be an explicit dependency here.